info/v1/container.go

```diff
@@ -922,13 +922,26 @@ type ProcessStats struct {
 	Ulimits []UlimitSpec `json:"ulimits,omitempty"`
 }
 
+type PerfError struct {
+	// Name of event which caused raising an error
+	EventName string `json:"event_name"`
+
+	// Action which caused raising an error i.e. opening perf event
+	Action string `json:"action"`
+	// Error code returned by calling a function
```

Comment:
add an here empty line to keep the style, other elements in stucts are separeted through empty line